### ISSUE TITLE
Stdout from linuxkit not having EOF

### ISSUE THREAD

**Issue by jayakumarn** [linuxkit/linuxkit#3200]
Currently planning to  use linuxkit for my CD pipeline . We have created the automated workflow for running linuxkit . The problem is when I am using the `"linuxkit run qemu -publish 8080:80 linuxkit "` 
stdout  of the above command is not having any EOF so my SSH component in my workflow is not stopping , even I tried to remove getty service from my yml file but still it doesnt help . Any other  clue ??

``

**Comment by justincormack** [comment]
Yes `qemu` is a bit annoying as it uses a tty and will not exit. If you look at the way we run the tests under https://github.com/linuxkit/linuxkit/tree/master/test/cases we try to make the LinuxKit image test itself, and shut down if it detects the test has completed, then we read the output for "test PASSED" in most cases. Otherwise you can test in parallel and send a SIGTERM to qemu.
